docs/cmdline-opts/sasl-authzid.d

```diff
@@ -0,0 +1,5 @@
+Long: sasl-authzid
+Help: Use this identity during SASL PLAIN authentication
+Added: 7.65.0
+---
+Use this identity during SASL PLAIN authentication.
```

Comment:
I think this could use a little expanding as if you're not a SASL PLAIN expert, this is really hard to understand.

Reply:
I have expanded this - could you please take a look and let me know what you think?

Reply:
:+1: